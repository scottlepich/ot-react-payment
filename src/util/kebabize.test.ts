import kebabize from "./kebabize";

describe("kebabize", () => {
  it("should convert camel case to kebab case", () => {
    expect(kebabize("camelCase")).toMatchSnapshot();
  });

  it("should convert pascal case to kebab case", () => {
    expect(kebabize("PascalCase")).toMatchSnapshot();
  });

  it("should not change kebab case", () => {
    expect(kebabize("kebab-case")).toMatchSnapshot();
  });
});
