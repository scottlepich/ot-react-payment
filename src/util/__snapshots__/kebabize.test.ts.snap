// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kebabize should convert camel case to kebab case 1`] = `"camel-case"`;

exports[`kebabize should convert pascal case to kebab case 1`] = `"pascal-case"`;

exports[`kebabize should not change kebab case 1`] = `"kebab-case"`;
